exporter/splunkhecexporter/README.md

```diff
@@ -28,6 +28,13 @@ The following configuration options can also be configured:
 - `max_content_length_logs` (default: 2097152): Maximum log data size in bytes per HTTP post limited to 2097152 bytes (2 MiB).
 - `splunk_app_name` (default: "OpenTelemetry Collector Contrib") App name is used to track telemetry information for Splunk App's using HEC by App name.
 - `splunk_app_version` (default: Current OpenTelemetry Collector Contrib Build Version): App version is used to track telemetry information for Splunk App's using HEC by App version.
+- `hec_fields/source` (default = 'com.splunk.source'): Specifies the mapping of a specific unified model attribute value to the standard source field of a HEC event.
+- `hec_fields/sourcetype` (default = 'com.splunk.sourcetype'): Specifies the mapping of a specific unified model attribute value to the standard sourcetype field of a HEC event.
+- `hec_fields/index` (default = 'com.splunk.index'):  Specifies the mapping of a specific unified model attribute value to the standard index field of a HEC event.
+- `hec_fields/host` (default = 'host.name'):  Specifies the mapping of a specific unified model attribute value to the standard host field and the `host.name` field of a HEC event.
+- `hec_fields/severity_text` (default = `otel.log.severity.text`): Specifies the name of the field to map the severity text field of log events.
+- `hec_fields/severity_number` (default = `otel.log.severity.number`): Specifies the name of the field to map the severity number field of log events.
+- `hec_fields/name` (default = `"otel.log.name`): Specifies the name of the field to map the name field of log events.
```

Comment:
Let's have two configs instead of one in exporter as well.

I'm thinking more about what naming would be the least confusing. Maybe these:
-`otel_attrs` or more explicit `hec_to_otel_attrs` for hec->otel mapping
- `hec_labels` or more explicit `otel_to_hec_labels` for otel->hec mapping
What do you think?

Reply:
Yes, that's what I meant to do. I goofed. I'll fix.